### PR TITLE
Add Portuguese translation 

### PR DESCRIPTION
**Pull request by alexsayal**
Hello,

The Portuguese team @con/portuguese sends the translated versions of the ultimate, GDPR, and DOA documents, following up #122.

These texts were already verified by independent back translation.

Let us know of any additional changes!

Best,

### CODE REVIEW

File: docs/source/gdpr/i18n/ultimate_gdpr.pt.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. _chap_consent_ultimate_gdpr_pt:
+
+Portuguese
+----------
+(translation courtesy of Ana Pina Rodrigues, Alexandre Sayal, Bruno Direito, Fernando Ferreira-Santos)
+
+**Version:** OBC-GDPR-ULT.pt 1.0.0
```
Comment (yarikoptic) [inline docs/source/gdpr/i18n/ultimate_gdpr.pt.rst:7]: please make sure you were translating the most recent version of the "ultimate" which already has version 

```
docs/source/gdpr/ultimate_gdpr.rst:**Version:** OBC-GDPR-ULT 1.1.0
```

**Review by yarikoptic — COMMENTED** [review]
This is great, thank you very much all for your contribution!  Left just 1 minor comment, and I will try to find yet another pt-speaking reviewer to make procedure fully kosher.

**Comment by yarikoptic** [comment]
Just to make sure -- I should alert all contributors to add everyone on @con/portuguese  to contributors, right?

File: docs/source/gdpr/i18n/ultimate_gdpr.pt.rst (hunk 1 above)
Comment (ferreira-santos) [reply docs/source/gdpr/i18n/ultimate_gdpr.pt.rst:7]: Added this to @con/portuguese -- will help any additional reviewers understand the process.

Comment (alexsayal) [reply docs/source/gdpr/i18n/ultimate_gdpr.pt.rst]: > please make sure you were translating the most recent version of the "ultimate" which already has version
> 
> ```
> docs/source/gdpr/ultimate_gdpr.rst:**Version:** OBC-GDPR-ULT 1.1.0
> ```

Thank you for this, we translated version 1.1.0. I have now corrected this typo.

**Comment by alexsayal** [comment]
> Just to make sure -- I should alert all contributors to add everyone on @con/portuguese to contributors, right?

Yes, all four members of @con/portuguese should be added as contributors! @ferreira-santos kindly explained the process in a comment above.

**Comment by yarikoptic** [comment]
ok, that didn't work, filed a https://github.com/all-contributors/app/issues/392 . I will flood this PR later (to not interfere) with instructions to @all-contributors to have everyone in @con/portuguese acknowledged.

**Comment by brunodireito** [comment]
Hi again @yarikoptic, 

Do you need any additional information?

Please let us know the current status of this process.

Thanks, 

Bruno

**Comment by yarikoptic** [comment]
hey @all-contributors please add @alexsayal alexsayal for translation

**Comment by yarikoptic** [comment]
hey @all-contributors please add @ferreira-santos for translation

**Comment by yarikoptic** [comment]
hey @all-contributors please add @brunodireito for translation

**Comment by yarikoptic** [comment]
I was hoping to find another reviewer but failed, so let's proceed as is. Thank you!

**Pull request merged**